_topic_map.yml

```diff
@@ -490,6 +490,8 @@ Topics:
   Topics:
   - Name: Troubleshooting installations
     File: troubleshooting-installations
+  - Name: Troubleshooting upgrades
+    File: troubleshooting-upgrades
```

Comment:
This is how  you add a file to the TOC in the left nav. The `File` element uses the file name of the assembly, minus the `.adoc` ending.